com/edocti/Connection.java

```diff
@@ -0,0 +1,10 @@
+package com.edocti.jintro;
```

Comment:
if package is com.edocti.jintro => folder structure must also be com/edocti/jintro (you need to move this class inside jintro folder)